recipe/meta.yaml

```diff
@@ -8,19 +8,20 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 553978badef3bb95af7557679f037a3f01cc167e9991fd9618578d3971cd62f6
+  sha256: 49a8375d72ba27efe7414a9ccc3ad5f969a13838bd089ad985ab836d03ffb8a4
 
 build:
   number: 0
-  # accelerate not yet available on OSX or Windows
-  skip: true  # [py<37 or osx or win]
+  skip: true  # [py<37]
+  skip: true  # [py>310 and ppc64le]
```

Comment:
quick note as to why now ppc64le is not supported would be appreciated.

Reply:
also in the commit message description.

Reply:
> `pytorch` isn't built on ppc for python 3.11, and `pytorch 2.0.x` isn't built for ppc at all.
Ok,  I understand why now. Nonetheless, a comment in the description would be nice.